### PR TITLE
init commit of idea journal, lesson 2

### PR DESCRIPTION
**Pull request by PrincAm**
This is an init version of Idea Journal app which we build together with attendees at lesson 2 - React fundamentals. See exercises in readme. Tasks are divided into three section - basic, state, props - and every section contains two exercises. The idea is that I will show how to make first and second will be on attendees. Whole app is located in /solution folder.
Presentation will be prepared separately.

### CODE REVIEW

File: lesson-2/idea-journal/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# React Fundamentals I
```
Comment (Aprillion) [inline lesson-2/idea-journal/README.md]: any reason for new file and not just edit `lesson-2/README.md`?

File: lesson-2/idea-journal/src/styles/NoteList.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.NoteList {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
```
Comment (Aprillion) [inline lesson-2/idea-journal/src/styles/NoteList.css]: missing new lines at the end of some css files :D

File: lesson-2/idea-journal/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import NewNoteModal from './NewNoteModal'
+import logo from '../logo.png'
+import '../styles/Header.css'
+
+class Header extends React.Component {
+
+	render() {
```
Comment (Aprillion) [inline lesson-2/idea-journal/src/components/Header.js:8]: using tabs in some files, spaces in other files?

File: lesson-2/idea-journal/src/components/NewNoteModal.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Modal from 'react-modal'
```
Comment (Aprillion) [inline lesson-2/idea-journal/src/components/NewNoteModal.js:2]: not sure if a Modal is Fundamental "1"...

**Review by Aprillion — APPROVED** [review]

File: lesson-2/idea-journal/README.md (hunk 1 above)
Comment (lspdv) [reply lesson-2/idea-journal/README.md]: yeah, please rewrite lesson-2/README.md rather, so every lesson keeps the same structure :-))) thanks!

**Review by lspdv — CHANGES_REQUESTED** [review]
please move README.md content to README.md in lesson-2 folder :-)

**Review by Aprillion — APPROVED** [review]

**Pull request merged**